circuit-test/src/main/kotlin/com/slack/circuit/test/FakeNavigator.kt

```diff
@@ -45,6 +51,9 @@ public class FakeNavigator : Navigator {
   /** Awaits the next [Screen] that was navigated to or throws if no screens were navigated to. */
   public suspend fun awaitNextScreen(): Screen = navigatedScreens.awaitItem()
 
+  /** Awaits the next navigation [resetRoot] or throws if no resets were performed. */
+  public suspend fun awaitReset(): Screen = newRoots.awaitItem()
```

Comment:
nit: update this name too (`awaitResetRoot()`)

Reply:
Done